### PR TITLE
Con 2935 feedback notification email

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs [hunk 1]
```diff
@@ -104,13 +104,39 @@ private async Task NotifyContact(Guid applicationId, int sequenceNo, Cancellatio
                 }
                 else if(sequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
                 {
-                    // TO DO: Send an email to notify organisation withdrawal approval to EPAO
                     var lastSubmission = application.ApplyData?.Apply.LatestOrganisationWithdrawalSubmission;
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs:107]: The code for both cases is almost the same - these could be combined with just the last submission part extracted

File: src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs [hunk 2]
```diff
@@ -104,13 +104,39 @@ private async Task NotifyContact(Guid applicationId, int sequenceNo, Cancellatio
...
+                    if (lastSubmission != null)
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs:108]: We should only be sending the email in the case of this being feedback and not for approval - there is switch above that check for that - similar can be used here

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs (hunk 1 above)
Comment (Howesy77) [reply src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs]: Updated code

File: src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs (hunk 2 above)
Comment (Howesy77) [reply src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs]: Done

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**